monorepo-migration/sync-ac.sh

```diff
@@ -77,12 +78,18 @@ function _remove_old_tags() {
 
 function _merge_histories() {
     cd "$SCRIPT_DIR"
-    git pull --no-edit --tags --allow-unrelated-histories --no-rebase --force "$REPO_PATH"
+    git pull --no-edit --tags --allow-unrelated-histories --no-rebase --force "$TMP_REPO_PATH"
     git commit --amend --message "$MERGE_COMMIT_MESSAGE"
 }
 
+function _move_files_into_subfolder() {
+    cd "$CURRENT_REPO_PATH"
+    git mv --force 'l10n.toml' "$REPO_NAME_TO_SYNC/l10n.toml"
+    git commit --message 'Fix l10n.toml file location'
```

Comment:
```suggestion
    git commit --message 'Move l10n.toml into android-components subfolder'
```
Maybe? I feel like we could appreciate the added clarity given our general concerns about the growing complexity of this script.

Reply:
Good call! Suggestion applied 🙂